downloader.sh

```diff
@@ -1,12 +1,68 @@
-echo "Enter Manga name:"
-read manga
-manga=${manga,,} # Converting it to lower case
-manga=${manga// /-} # Removing spaces and adding - in their places
-echo "Enter the chapter range."
-echo "Start:"
-read chaps
-echo "End:"
-read chape
+a="$#"
+declare -i m=0,c=0
+#checking for existence of -t , -chap in argument
+for (( i=1; i<=a; i++))
+do
+	if [ "${!i}" = "-t" ]; then
+		m=$i
+	fi
+	if [ "${!i}" = "-chap" ]; then
```

Comment:
Since its multiple character function, please make it --chap :) Sorry I forgot about this.